src/main.py

```diff
@@ -50,6 +51,27 @@
 if 'debug' not in config:
 	config['debug'] = debug
 
+# Setup event commands
```

Comment:
No. Empty string in python gets evaluated to `False`. (I only found out now myself)